x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_add_edit/hooks/use_monitor_not_found.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useEffect, useState } from 'react';
+import { EuiButton, EuiCallOut, EuiLink, EuiSpacer } from '@elastic/eui';
+import { IHttpFetchError, ResponseErrorBody } from '@kbn/core-http-browser';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useFetcher } from '@kbn/observability-plugin/public';
+import { MonitorNotFoundPage } from '../../monitor_details/monitor_not_found_page';
+import { useGetUrlParams, useUrlParams } from '../../../hooks';
+import { deletePackagePolicy } from '../../../state/monitor_management/api';
+
+export const useMonitorNotFound = (error?: IHttpFetchError<ResponseErrorBody>, id?: string) => {
```

Comment:
This should be a functional component instead of a hook as it returns React Node.